test/http_SUITE.erl

```diff
@@ -497,6 +497,10 @@ The document has moved
 		{400, "GET / HTTP/1.1\r\nHost: ninenines.eu\r\n\r\n"},
 		{400, "GET http://proxy/ HTTP/1.1\r\n\r\n"},
 		{400, "GET / HTTP/1.1\r\nHost: localhost:bad_port\r\n\r\n"},
+        {400, "GET foo HTTP/1.1\r\n"},
+        {400, "GET foo\r\n"},
+        {400, "GET HTTP/1.1\r\n"},
+        {400, "GET HTTP/1.1 HTTP/1.1\r\n"},
 		{505, ResponsePacket},
```

Comment:
This testcase fails, but has been failing before this PR. Changing `505` to `400` passes the test but I'm not sure if that's the right behaviour.